docs/build_setup.rst

```diff
@@ -0,0 +1,22 @@
+.. _build_setup:
+
+Build Setup
+===========
+
+#. 
+   Install Intel(R) oneAPI DPC++ Compiler (select variant as per requirement).
```

Comment:
Should we link to `selecting_a_compiler.rst` here?

Reply:
Done; thanks.